authorization/provider/config.go

```diff
@@ -0,0 +1,34 @@
+package provider
+
+import (
+	"fmt"
+
+	"github.com/plgd-dev/cloud/authorization/oauth"
+	"github.com/plgd-dev/cloud/pkg/net/http/client"
+)
+
+// Config general configuration
+type Config struct {
+	Provider     string `yaml:"provider" json:"provider" default:"generic"` // value which comes from the device during the sign-up ("apn")
```

Comment:
is it? what is the property then for configuring providerType?

Reply:
providerType = provider